model/riscv_sys_regs.sail

```diff
@@ -560,11 +607,11 @@ register mtime : bits(64)
 register minstret : bits(64)
 register minstret_written : bool
 
-function retire_instruction() -> unit = {
+function retire_instruction(orig_priv : Privilege) -> unit = {
   if   minstret_written == true
   then minstret_written = false
   else if mcountinhibit.IR() == 0b0
```

Comment:
```suggestion
  else if should_inc_minstret(priv)
```
and move the check for mcountinhibit.IR() into should_inc_minstret?

Reply:
Yeah, I said as much in my review for tick_clock

Reply:
Done